__tests__/html/focusManagement.disableHeroCard.obsolete.html

```diff
@@ -15,14 +15,29 @@
     <script type="text/babel" data-presets="env,stage-3,react">
       const {
         WebChat: {
-          Components: { HeroCardContent }
+          Components: { HeroCardContent },
+          hooks: { useActivities }
         }
       } = window;
 
       run(async function () {
         const store = testHelpers.createStore();
         const directLine = WebChat.createDirectLine({ token: await testHelpers.token.fetchDirectLineToken() });
 
+        const AutoDisableHeroCardContent = ({ activity, attachment }) => {
+          const [activities] = useActivities();
```

Comment:
This is the correct way, use hooks to detect if the activity is the most recent one.

We should not relies on middleware because it could be skipped due to caching.